samples/lib/widgets/popover_menu.dart

```diff
@@ -31,9 +31,10 @@ class _State extends StatefulSampleState<PopoverMenuPage> with SingleTickerProvi
         title: const Text('Edit Notes'),
         actions: [
           FPopoverMenu(
-            controller: controller,
+            popoverController: controller,
             menuAnchor: Alignment.topRight,
             childAnchor: Alignment.bottomRight,
+            maxHeight: 200,
```

Comment:
I don't think popover should generally be scrollable. It's ok to add the option in, but we shouldn't be promoting it.

On the other hand, `FSelectMenuTile` example should be scrollable. That's the main use case of why tiles need a scroll in the first place.